.github/workflows/test.yml

```
@@ -27,7 +24,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: "${{ env.NODE }}"
+          node-version: 16
```

For what it's worth, I believe this should always install the latest LTS release:

```suggestion
          node-version: lts/*
```